docker/observability/services/prometheus.yml

```diff
@@ -1,15 +1,22 @@
+x-common-extra-hosts: &common-extra-hosts
+  extra_hosts:
+    - 'agora-gene-api:host-gateway'
```

Comment:
This allows Prometheus to scrape metrics from the agora-gene-api server when it is run using its development server (without Docker).